### DISCUSSION TITLE
Visualize raw request in jetclient

### DISCUSSION THREAD

**Discussion by tiboleemans** [JetClient/jet-client-support#39] [Q&A] [answered]
Is it possible to see the raw request that has been sent in jetclient? 

Like in postman you can open the console and see all the details of the request.

**Comment by AntonShuvaev** [comment, marked as answer]
To view the raw request, simply click on "Generate Code Snippet" (shortcut: Alt+Shift+Cmd+G) from the dropdown menu located near the "Send" button. Then, select "HTTP" from the dropdown options. This will display the raw HTTP request.
